ci-operator/config/ComplianceAsCode/content/ComplianceAsCode-content-master__4.14.yaml

```diff
@@ -14,7 +14,7 @@ releases:
       architecture: arm64
       product: ocp
       stream: nightly
-      version: "4.18"
+      version: "4.14"
```

Comment:
This accidentally slipped in https://github.com/openshift/release/pull/62801/files#diff-03622b08999443dbc85cd06d56c484e0f11dc3406b7171dcdf56105be79c619dR17